### DISCUSSION TITLE
Styling RecordsList.php

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16209] [How to]
Trying to improve PDF by injecting own custom inline css to RecordsList.php, but it not coming into effect (even with !important)
Plus: $html .  have class "products-table" to which I cannot find reference in any styles files. Where this class is set?

Thanks for help

**Comment by s2b-git** [comment]
As I answered in other thread, i use $(recordsList to pull records to PDF report
Customization of table, and content:
![image](https://user-images.githubusercontent.com/84110093/175774904-058a6452-f02d-4c61-88eb-2b21e2519da8.png)

**Reply by mariuszkrzaczkowski** [reply]
include your HTML code with styles and describe what effect you want to achieve

**Comment by mariuszkrzaczkowski** [comment]
style block is available in the new version
![image](https://user-images.githubusercontent.com/10198654/175789305-60932edb-6066-4c7e-a293-c50fdf084dd7.png)
there will also be a new chrome engine for generating PDF

**Reply by s2b-git** [reply]
This is nice :-) 
However I still looking for a way to override /vendor/yetiforce/yetiforcepdf/lib/Style/Style.php
Any changes made in RecordsList.php are not respected. What would be the best approach?

Thank you
